c/experimental/stf/test/test_places.cpp

```
@@ -233,60 +231,340 @@ C2H_TEST("composite data place with stf_exec_place_grid_create (vector of places
+
+  stf_data_place_handle composite_dplace = stf_data_place_composite(grid, blocked_mapper_1d);
+  REQUIRE(composite_dplace != nullptr);
+  stf_exec_place_set_affine_data_place(grid, composite_dplace);
+
+  stf_ctx_handle ctx = stf_ctx_create();
+  REQUIRE(ctx != nullptr);
+
+  float* X = static_cast<float*>(malloc(4 * sizeof(float)));
+  for (size_t i = 0; i < 4; ++i)
+  {
+    X[i] = 0.0f;
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)